isis/src/base/objs/PvlObject/PvlObject.cpp

```diff
@@ -59,6 +59,49 @@ namespace Isis {
   }
 
 
+ 
+  /**
+   * Creates a PvlObject with a specified name converted 
+   * from a json object.
+   *
+   * TODO: Needs to be more robust, right now exists in order to 
+   * convert an Ale ISD NaifKeywords group over to an ISIS 
+   * compatible version. 
```

Comment:
Was this handled or is there more that needs to happen here?

Reply:
It seems to work so far, simply doesn't handle all types. Probably something that should change in the near future. Not sure if super important right now,

Reply:
I think this is one of those things were we can expand the functionality as needed. I would remove this TODO and instead document which types it converts right now.